src/symbols/analyzer.rs

```diff
@@ -115,10 +116,22 @@ impl<'a> RootSymbol<'a> {
       .flat_map(|d| d.into_definitions())
   }
 
+  /// Goes to the definitions of the specified symbol.
+  pub fn go_to_definitions_or_unresolveds<'b>(
```

Comment:
Is it a typo? It reads strange :/

Reply:
I could maybe name this `go_to_definition_or_unresolved_items`. Not sure.